images/logging-fluentd.yml

```diff
@@ -9,13 +9,12 @@ content:
 dependents:
 - cluster-logging-operator
```

Comment:
Technically this would need a new dist-git branch and be added to the rhel-8 tag package list, with the following lines added:
```
distgit:
  branch: rhaos-{MAJOR}.{MINOR}-rhel-8
```

Reply:
Done. Thanks!